src/Pact/Analyze/Translate.hs

```diff
@@ -236,6 +238,10 @@ data TranslateState
       -- correspond to successful exit early due to the lack of a failure.
       -- These three "success" edges all join together at the same vertex.
     , _tsFoundVars     :: [(VarId, Text, EType)]
+
+      -- Vars representing nondeterministic choice between two branches. These
+      -- are used for continuing on or rolling back in evaluation of pacts.
+    , _tsNondets       :: ![VarId]
```

Comment:
same here.. "step [is] taken"? "step [is] run"? "pact continues"? `_tsStepNondets`? `_tsPactNondets`?